autorest/adal/config.go

```diff
@@ -89,3 +93,58 @@ func NewOAuthConfigWithAPIVersion(activeDirectoryEndpoint, tenantID string, apiV
 		DeviceCodeEndpoint: *deviceCodeURL,
 	}, nil
 }
+
+// MultiTenantOAuthConfig provides endpoints for primary and aulixiary tenant IDs.
+type MultiTenantOAuthConfig interface {
+	PrimaryTenant() *OAuthConfig
+	AuxiliaryTenants() []*OAuthConfig
+}
+
+// Config contains optional OAuthConfig creation arguments.
+type Config struct {
```

Comment:
>Config [](start = 5, length = 6)

well is this meant to be just the config for AAD or more generic ?

Reply:
It's meant as optional config params for `NewMultiTenantOAuthConfig()` so that we don't end up in the same situation as `NewOAuthConfig()` and `NewOAuthConfigWithAPIVersion()`.

Reply:
Maybe `Options`?